src/test/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloudIT.java

```diff
@@ -231,6 +232,21 @@ public void test1WorkerCreatedFor2Executors() throws Exception {
         assertEquals(logs(), 1, planned.size());
     }
 
+    @Test(timeout = 300000)
+    public void testMultipleLabels() throws Exception {
+        // For a configuration with multiple labels, test if job label matches one of the configuration's labels
+        logOutput.reset();
+
+        ComputeEngineCloud cloud = (ComputeEngineCloud) r.jenkins.clouds.get(0);
+        InstanceConfiguration ic = instanceConfiguration(DEB_JAVA_STARTUP_SCRIPT, NUM_EXECUTORS, MULTIPLE_LABEL);
+        cloud.addConfiguration(ic);
+        // Add a new node
+        Collection<NodeProvisioner.PlannedNode> planned = cloud.provision(new LabelAtom(LABEL), 1);
+
+        // There should be a planned node
```

Comment:
Should probably also check that the created node has the expected labels as well, similar to testWorkerCreated()

Reply:
The test in testWorkerCreated for label actually refers to a label that associates it with the ComputeEngineCloud, so I'm not sure it's applicable here.

Reply:
Good point. Digging a little deeper, it looks like you can get the provisioned node from the planned node's future, and then verify the node's label string.